### PR TITLE
Update dependencies / move to github actions

### PR DESCRIPTION
**Pull request by Roaders**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14.x'
+    - run: npm ci
+    - uses: JS-DevTools/npm-publish@v1
+      with:
+        token: ${{ secrets.NPM_TOKEN }}
```
Comment (Davidhanson90) [inline .github/workflows/release.yml]: have you generated a new token for this?

Comment (Roaders) [reply .github/workflows/release.yml]: no, it'll use the same token that was in there already

Comment (Roaders) [reply .github/workflows/release.yml]: ah... no it won't. there wasn't one!

Comment (Roaders) [reply .github/workflows/release.yml]: we also need to replace the github pages step

Comment (Davidhanson90) [reply .github/workflows/release.yml]: @bingenito  recommends we create a new token.

Comment (Roaders) [reply .github/workflows/release.yml]: ok, secret added and github pages added

**Review by Davidhanson90 — APPROVED** [review]

**Pull request merged**